Test/TrendsCalculatorTest.cs

```diff
@@ -0,0 +1,193 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using TrendsCalculator.Library.Core.Strategy;
+
+namespace TrendsCalculator.Test
+{
+    [TestClass]
+    public class TrendsCalculatorTest
+    {
+        [TestMethod]
+        public void TrendsCalculatorTest_CustomCriteria_Suite1()
+        {
+            var trendingCalculator = new Library.TrendsCalculator(TrendCalculationStrategy.Custom);
+            var inputData = DataPreparator.PrepareData_Suite1();
```

Comment:
Instead of naming suite1 in the name: Give an intuitive name of what the test case is about. It is Ok if the method name gets a little long.. E.g.You need not write "TrendsCalculatorTest" as this is implicit.
E.g. Test_CustomCritera_BlankRecords_ShouldGiveException
Test_CustomCriteria_InsufficientDemand_ShouldMakeElementMostTrending